client/src/components/Header.jsx

```diff
@@ -2,13 +2,24 @@ import React from 'react';
 import $ from 'jquery';
 import styles from '../styles/Header.css';
 
+require('../../../server/helpers/jquery-visible.min.js');
+
 class Header extends React.Component {
   constructor(props) {
     super(props);
   }
 
-  renderRatingHistogram(reviews) {
+  componentDidMount() {
+    // check if inview and if so animate
+    setInterval(() => {
+      if ($('#scrollTarget').visible()) {
+        // wildcard jquery selector
+        $('[id^=histBar]').addClass(styles.animated);
```

Comment:
this looks cool. would love to hear about what this is later!

Reply:
this is jquery to detect if target is in viewport and add class to animate histogram